From efedbf9080c19241c2aa9ee7ba901245d38c8fa2 Mon Sep 17 00:00:00 2001
From: Naveen Saini <naveen.kumar.saini@intel.com>
Date: Mon, 7 Mar 2022 16:44:24 +0800
Subject: [PATCH] CMakeLists.txt: exclude host system headers

Upstream-Status: Inappropriate

Signed-off-by: Naveen Saini <naveen.kumar.saini@intel.com>
---
diff --git a/sources/ippcp/crypto_mb/CMakeLists.txt b/sources/ippcp/crypto_mb/CMakeLists.txt
index 0a8727a..4772e99 100644
--- a/sources/ippcp/crypto_mb/CMakeLists.txt
+++ b/sources/ippcp/crypto_mb/CMakeLists.txt
@@ -197,7 +197,6 @@ include_directories(
     ${INTERNAL_INCLUDE_DIR}
     ${INTERNAL_INCLUDE_DIR}/single_cpu/crypto_mb
     $<$<C_COMPILER_ID:Intel>:$ENV{ROOT}/compiler/include $ENV{ROOT}/compiler/include/icc>
-    $<$<NOT:$<C_COMPILER_ID:Intel>>:${CMAKE_SYSTEM_INCLUDE_PATH}>
     $<$<AND:$<NOT:$<C_COMPILER_ID:IntelLLVM>>,$<BOOL:${MSVC_IDE}>>:$ENV{INCLUDE}>
 )
 
